src/pages/AnswerPage.jsx

```diff
@@ -67,19 +67,13 @@ const AnswerPage = () => {
           {isDeleting ? "삭제중" : "삭제하기"}
         </Button>
         <QuestionBox count={questionCount}>
-          <ul className="tablet:gap-20 mt-16 flex w-full flex-col gap-16">
-            {questionList.map((question) => {
-              return (
-                <li key={question.id}>
-                  <FeedCard
-                    isAnswerPage={true}
-                    subject={subject}
-                    question={question}
-                  />
-                </li>
-              );
-            })}
-          </ul>
+          <QuestionList
+            isInitialLoading={isInitialLoading}
+            questionList={questionList}
+            subject={subject}
+            isLoading={isLoading}
+            isAnswerPage={true}
+          />
```

Comment:
```suggestion
          <QuestionList
            isInitialLoading={isInitialLoading}
            isLoading={isLoading}
            questionList={questionList}
            subject={subject}
            isAnswerPage={true}
          />
```
[제안] 이렇게 순서를 바꾸면 좀 더 인식하는 데에 쉽지 않을까 생각이 듭니다